content/en/tracing/setup_overview/open_standards/_index.md

```diff
@@ -2,9 +2,98 @@
 
 {{< partial name="apm/apm-opentracing.html" >}}
 
+<br>
+
+## OpenTelemetry Collector
+
+Alternatively, Datadog has [an exporter that uses the OpenTelemetry Collector][3] to receive traces and metrics data from the OpenTelemetry SDKs, and to forward the data on to Datadog (without the Agent). It works with all supported languages, and you configure it by supplying the [Datadog API key][4]:
+
+```
+datadog:
+  api:
+    key: "<API key>"
+```
+To send the data to the Datadog EU site, also set the `site` parameter:
+```
+datadog:
+  api:
+    key: "<API key>"
+    site: datadoghq.eu
+```
+
+On each OpenTelemetry-instrumented application, set the resource attributes `development.environment`, `service.name`, and `service.version` using the language's SDK. As a fall-back, you can also configure hostname (optionally), environment, service name, and service version at the collector level for unified service tagging by following the [example configuration file][5]. If you don't specify the hostname explicitly, the exporter attempts to get an automatic default by checking the following sources in order, falling back to the next one if the current one is unavailable or invalid:
+
+<!--- 1. Hostname set by another OpenTelemetry component -->
+1. Manually set hostname in configuration
+1. EC2 non-default hostname (if in EC2 instance)
+1. EC2 instance id (if in EC2 instance)
+1. Fully qualified domain name
+1. Operating system host name
+
+### Ingesting OpenTelemetry Traces with the Collector
+
+The exporter assumes you have a pipeline that not only uses the exporter, but also includes a batch processor configured with the following:
+  - A `timeout` setting of `10s` (10 seconds). A batch representing 10 seconds of traces is a constraint of Datadog's API Intake for Trace Related Statistics. Without this setting, trace related metrics including `.hits`, `.errors`, and `.duration` for different services and service resources may be inaccurate over periods of time.
+
+<div class="alert alert-warning">
+The Datadog exporter for the OpenTelemetry Collector is currently in beta. It may consume high CPU and memory resources. Configuring particularly the pipeline and batch processor may take some iteration before it responds with accurate metrics given your production environment. <a href="https://docs.datadoghq.com/help/">Reach out to support</a> if it doesn't work as you expect.
+</div>
+
+Here is an example pipeline:
+
+```
+receivers:
+  examplereceiver:
+
+processors:
+  batch:
+    timeout: 10s
+
+exporters:
+  datadog/api:
+    hostname: customhostname
+    env: prod
+    service: myservice
+    version: myversion
+
+    tags:
+      - example:tag
+
+    api:
+      key: aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
+      site: datadoghq.eu
+
+service:
+  pipelines:
+    traces:
+      receivers: [examplereceiver]
+      processors: [batch]
+      exporters: [datadog/api]
+```
+
+## Further Reading
+
+{{< partial name="whats-next/whats-next.html" >}}
+
```

Comment:
I realise now i inlined a lot of markdown links that belong as [] thingys instead. sorry about that.

Reply:
lol, nah, all good. I appreciate getting the good info and can take care of the formatting.